vllm/worker/worker.py

```diff
@@ -229,6 +231,9 @@ def determine_num_available_blocks(self) -> Tuple[int, int]:
         num_gpu_blocks = max(num_gpu_blocks, 0)
         num_cpu_blocks = max(num_cpu_blocks, 0)
 
+        end_time = time.time()
+        logger.info("Memory profiling took %.2f seconds",
+                    end_time - start_time)
         logger.info(
             "Memory profiling results: total_gpu_memory=%.2fGiB"
```

Comment:
Let's keep the number of logs down - can you add time to the following log? Such as `duration=%.2f seconds`

It would be nice to make each item comma separated rather than with spaces

Reply:
good point, fixed in [a545b67](https://github.com/vllm-project/vllm/pull/10460/commits/a545b677e75033ac97ee107439811da43dfd8be6)